Content.Shared/Actions/SharedActionsSystem.cs

```diff
@@ -370,6 +371,13 @@ private void OnActionRequest(RequestPerformActionEvent ev, EntitySessionEventArg
 
         BaseActionEvent? performEvent = null;
 
+        if (action.CheckMobState != null)
+        {
+            if (!TryComp<MobStateComponent>(user, out var mobState) ||
+                mobState.CurrentState != action.CheckMobState)
+                return;
+        }
+
```

Comment:
Could this just use ActionBlocker instead?

Reply:
I guess so. I just realized it also needs to check if someone is sleeping. I'll just make a new check named IsConcious or something.